src/poliastro/tests/test_plotting2d.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+from unittest import mock
+
+from astropy import units as u
+
+from poliastro.examples import iss
+from poliastro.plotting import OrbitPlotter2D
+
+
+def test_get_figure_has_expected_properties():
+    frame = OrbitPlotter2D()
+    figure = frame.figure
+
+    assert figure["data"] == [{}]
+    assert figure["layout"]["autosize"] is True
+    assert "xaxis" in figure["layout"]["scene"]
+    assert "yaxis" in figure["layout"]["scene"]
+    assert "aspectmode" in figure["layout"]["scene"]
+
+
+def test_set_different_attractor_raises_error():
+    body1 = mock.MagicMock()
+    body1.name = "1"
+
+    body2 = mock.MagicMock()
+    body2.name = "2"
+
+    frame = OrbitPlotter2D()
+    frame.set_attractor(body1)
+
+    with pytest.raises(NotImplementedError) as excinfo:
+        frame.set_attractor(body2)
+    assert "Attractor has already been set to 1." in excinfo.exconly()
+
+
+def test_plot_sets_attractor():
+    frame = OrbitPlotter2D()
+    assert frame._attractor is None
```

Comment:
Not sure whether we should test against private properties, but on the other hand I don't have a better idea :confused:

Reply:
What is your suggestion?

Reply:
Don't have a better suggestion really, just leave it like it is.

Reply:
Okay!